content/intro-to-storybook/react-native/en/composite-component.md

```diff
@@ -0,0 +1,326 @@
+---
+title: 'Assemble a composite component'
+tocTitle: 'Composite component'
+description: 'Assemble a composite component out of simpler components'
+commit: '8db511e'
```

Comment:
We should remove these `commit`s if they don't exist for this version

Reply:
i checked and the frontmatter commit key was removed